### ISSUE TITLE
Support Chinese Input

### ISSUE THREAD

**Issue by ghost** [futo-org/android-keyboard#692]
### Use Cases

> What is the simplified pinyin mode? How to use the simplified pinyin mode?
Google Input supports simplified input, such as inputting "wlj" to get "王老吉". In addition, in order to meet the needs of network abbreviations, Google Input also supports simplified initials of zh, ch, and shi. For example, if you want to input "real-time query", you can input "ssry" or "shshry" to get "everything goes as you wish". If you want to use this feature, please click the "Settings" button on the status bar, select "Property Settings" > "Pinyin Mode", select "Use "z,c,s" for "zh,ch,sh" when abbreviating" to enable this feature. At the same time, Google Input Method also supports mixed input of abbreviated and full pinyin, for example: input "srf" "sruf" "shrfa" can get "input method". pinyin_62660_cn.gif Please note: Here, the role of the initial consonant abbreviation is the same as the "z, s, c" in the fuzzy sound. However, these are two different functions. Even if you do not select the fuzzy sound in the settings, you can still use "zly" to input "Zhang Liangying". Effective use of the initial consonant abbreviation can improve input efficiency and reduce mistypes. For example, to input the words "转瞬即逝", if you enter the traditional initial consonant abbreviation, you must enter "zhshjsh", so you need to enter 3 initial consonants including "h", which is easy to cause input errors. If you use the abbreviation mode, you only need to enter "zsjs". https://support.google.com/pinyin/answer/62660

### References

chinese 8 auxiliary codes to choose from, 7 pinyin presets, extremely suitable for software development

https://github.com/amzxyz/rime_wanxiang_pinyin
https://github.com/gaboolic/rime-shuangpin-fuzhuma

**Comment by Mieridduryn** [comment]
I would also like Chinese input, as someone who is learning (and has been learnint) Mandarin as a second language. It'd be especially useful in the wake of the migration from TikTok to XiaoHongShu.

**Comment by KatieTheDev** [comment]
I agree, this would be a nice addition. I want to learn Mandarin, but this is made harder by a lack of FOSS keyboard options.

**Comment by Liu8Can** [comment]
The recognition of Chinese Simplified Chinese is too poor, and it will always be recognized as Chinese Traditional

**Comment by XingShelly** [comment]
I suggest working with Kuaizi IME team:
https://github.com/crazydan-studio/kuaizi-ime

 They have come up with what I have been calling for, for years. A Chinese input method that makes sense for Chinese only. QWERTY keyboards are really stupid for Chinese, as most buttons do not get used most of the time. The keyboard that they came up with is very usable, but there are still many kinks to work out, and the layout still needs a little bit more modification; I also disagree with their stance that using prediction is wrong and you have to remember the characters yourself, but the core idea makes SO MUCH SENSE!

Please work with them and try to integrate their method. This is the most logical method to implement Chinese.

Another team that already made a working Chinese keyboard with prediction (although it's QWERTY) is the fcitx android team:
https://github.com/fcitx5-android/fcitx5-android/

Their keyboard just works. No swiping, like other Chinese keyboards, but it works.

Instead of each team working on its own and reinventing the wheel, I think we should gather our resources together and work with each other. IDK

**Comment by Zvonimir-FUTO** [comment]
Will be resolved through #707
